spec.md

```diff
@@ -23,7 +23,10 @@ If a `<script type="importmap">` is encountered when _acquiring import maps_ is
 Encountering a `<script type="importmap">` while _acquiring import maps_ is true will kick off a procedure roughly like this:
 
 1. If it's an external script (i.e. has a `src=""`), fetch it, using the usual "good defaults" of `<script type="module">`. (E.g., always UTF-8, "cors" mode, MIME type must match, ...)
-  - What should be the MIME type? Let's say `application/json+importmap`? Maybe accept any JSON MIME type.
+   - What should be the MIME type? Let's say `application/json+importmap`.
```

Comment:
It seems that the convention for MIME-type extensions would suggest `application/importmap+json` rather than `application/json+importmap` from skimming through https://www.iana.org/assignments/media-types/media-types.xhtml ^^

Reply:
@Malvoz, thanks for pointing that out.  I see many `...+xml` but no `...xml+...` that isn't also `...+xml`.  Fixed.